tests/conftest.py

```diff
@@ -545,14 +546,19 @@ def text_log_error_lines(test_job, failure_lines):
 
 @pytest.fixture
 def test_perf_alert_summary(test_repository, push_stored, test_perf_framework, test_issue_tracker):
-    return PerformanceAlertSummary.objects.create(
+    test_perf_tag = PerformanceTag.objects.create(id=1, name='harness')
```

Comment:
Please use `performance_alert_summary.id` instead of `1`

Reply:
Could the code work without mentioning those ids? The id is set to be AutoField and maybe it is not necessary to specify it.

Reply:
ignore my comment @esanuandra  is correct: you should be able to make the tag without setting the `id`   that would explain why it is not showing up in the extract.